cellpack/autopack/Ingredient.py

```diff
@@ -3329,14 +3423,28 @@ def checkSphCollisions(
                 ):  # point in region of influence
                     # need to update the distances of the master grid with new smaller distance
                     if pt in newDistPoints:
-                        if signed_distance_to_sphere_surface < newDistPoints[pt]:
-                            newDistPoints[pt] = signed_distance_to_sphere_surface
+                        newDistPoints[pt] = min(
+                            signed_distance_to_sphere_surface, newDistPoints[pt]
+                        )
                     else:
                         newDistPoints[pt] = signed_distance_to_sphere_surface
-                        # sphNum += 1
-                else:
-                    continue
-
+            if not at_max_level:
+                # we didn't find any colisions with the this level, but we still want
+                # the inside points to be based on the most detailed geom
+                new_level = self.maxLevel
+                nxtLevelSpheres = self.positions[new_level]
+                nxtLevelRadii = self.radii[new_level]
+                return self.checkSphCollisions(
```

Comment:
This jumps us all the way down to the more detailed geom to calculate the inside points. Before if no collisions had been found, the inside points would be based on the encapsulating radius.